README.md

```diff
@@ -1,10 +1,10 @@
 # korge-compose
 
-KProject adding support for JetPack compose composables to KorGE
+KProject adding support for [JetPack compose](https://www.jetbrains.com/lp/compose-mpp/) composables to KorGE and Korlibs
```

Comment:
I recently was aware that this should be Compose Multiplatform instead of JetPack Compose. Can you change it too?

Reply:
Done